lan865x/README.md

```diff
@@ -0,0 +1,71 @@
+
+## ESP-IDF Usage
+
+> [!TIP]
+> Use [Component for Ethernet Initialization](https://components.espressif.com/components/espressif/ethernet_init) for smoother user experience.
+
+Add this component from [IDF Component Manager](https://components.espressif.com/) to your project using `idf.py add-dependency` and include `esp_eth_mac_lan865x.h` and `esp_eth_phy_lan865x.h`,
+
+```c
+#include "esp_eth_mac_lan865x.h"
+#include "esp_eth_phy_lan867x.h"
+```
+
+initialize SPI bus, and create `mac` and `phy` driver instance by calling `esp_eth_mac_new_lan865x` and `esp_eth_phy_new_lan865x()`
+
+```c
+// Install GPIO interrupt service (as the SPI-Ethernet module is interrupt-driven)
+gpio_install_isr_service(0);
+// SPI bus configuration
+spi_device_handle_t spi_handle = NULL;
+spi_bus_config_t buscfg = {
+    .miso_io_num = CONFIG_EXAMPLE_ETH_SPI_MISO_GPIO,
+    .mosi_io_num = CONFIG_EXAMPLE_ETH_SPI_MOSI_GPIO,
+    .sclk_io_num = CONFIG_EXAMPLE_ETH_SPI_SCLK_GPIO,
+    .quadwp_io_num = -1,
+    .quadhd_io_num = -1,
+};
+ESP_ERROR_CHECK(spi_bus_initialize(CONFIG_EXAMPLE_ETH_SPI_HOST, &buscfg, 1));
+// Configure SPI device
+spi_device_interface_config_t spi_devcfg = {
+    .mode = 0,
+    .clock_speed_hz = CONFIG_EXAMPLE_ETH_SPI_CLOCK_MHZ * 1000 * 1000,
+    .spics_io_num = CONFIG_EXAMPLE_ETH_SPI_CS_GPIO,
+    .queue_size = 20
+};
+
+eth_mac_config_t mac_config = ETH_MAC_DEFAULT_CONFIG();
+eth_phy_config_t phy_config = ETH_PHY_DEFAULT_CONFIG();
+
+eth_lan865x_config_t lan865x_config = ETH_LAN865X_DEFAULT_CONFIG(CONFIG_EXAMPLE_ETH_SPI_HOST, &spi_devcfg);
+
+dev_out->mac = esp_eth_mac_new_lan865x(&lan865x_config, &mac_config);
+dev_out->phy = esp_eth_phy_new_lan865x(&phy_config);
+```
+
+## Collision avoidance methods: CSMA/CD and PLCA
+
+In most cases you can rely on the default CSMA/CD (Carrier Sense Multiple Access / Collision Detection) provided by the MAC layer to avoid collisions when connecting multiple nodes. It requires zero configuration to work and does a good job of coordinating when data is sent. However, due to its nature, the delays between transmissions are random. For timing-critical applications, deterministic PLCA (PHY-Level Collision Avoidance) provides a guaranteed time period between transmissions along with improved throughput. Read more about PLCA and its usage at [ieee802.org](https://www.ieee802.org/3/cg/public/July2018/PLCA%20FAQ.pdf).
+
+### Configuring PLCA
+
+PLCA configuration is performed using the `esp_eth_ioctl` function with one of the commands described in [LAN86XX PHY Sub-component](../lan86xx_common/README.md).
```

Comment:
>  [LAN86XX PHY Sub-component](../lan86xx_common/README.md).

Is this link accessible when deployed as a component (two components)?

Reply:
Probably not :) Do you have any suggestion how address such situations?

Reply:
I will resolve in the future.